### DISCUSSION TITLE
SessionMiddleware maybe document security related impact of max_age = None

### DISCUSSION THREAD

**Discussion by bastianh** [encode/starlette#1850] [Potential Issue]
Hi,

I noticed in `SessionMiddleware` it's possible to set `max_age` to `None` to only make the session last as long as the browser is open. I think it should be made clearer in the documentation that it also makes the session to be signed for an infinite time, because someone might think that it's more secure to have the cookie deleted when the browser is closed when technically it might be more insecure because a stolen cookie could be used for an infinite time.

Bastian

**Comment by Kludex** [comment]
PR always welcome to improve documentation.

**Reply by euri10** [reply]
sounds contradictory with https://github.com/encode/starlette/pull/1387 @bastianh

**Reply by euri10** [reply]
mm I misread, though you were implying the cookie was not deleted on browser closing :) /hide
